contributed_definitions/NXcontainer.nxdl.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+# 
+# Copyright (C) 2008-2016 NeXus International Advisory Committee (NIAC)
+# 
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+	name="NXcontainer"
+	category="contributed"
+	version="1.0"
+	type="group"
+	extends="NXobject"
+	>
+
+	<doc>
+		State of a container holding the sample under investigation.
+		
+		A container is any object in the beam path which absorbs the beam and 
+		whose contribution to the overall attenuation/scattering needs to be 
+		determined to process the experimental data. Examples of containers 
+		include glass capillary tubes, vanadium cans, windows in furnaces or 
+		diamonds in a Diamond Anvil Cell. The following figures show a complex
+		example of a container:
+		
+		.. figure:: container/ComplexExampleContainer.png
+		   
+		   A hypothetical capillary furnace. The beam passes from left to right
+		   (blue dashes), passing through window 1, then window 2, before 
+		   passing through the downstream wall of the capillary. It is then 
+		   scattered by the sample with scattered beams passing through the 
+		   upstream wall of the capillary, then windows 4 and 5. As part of the
+		   corrections for a PDF experiment it is necessary to subtract the PDF
+		   of the empty container (i.e. each of the windows and the capillary).
+		   To calculate the PDF of the empty container it is necessary to have 
+		   the measured scattering data and to know the nature (e.g. density, 
+		   elemental composition, etc.) of the portion of the container which 
+		   the beam passed through.
+		   
+		   figure:: container/ComplexContainerBeampath.png
```

Comment:
this line does not generate the figure yet
prefix with two dots at the correct indentation, like line 44 above, then the documentation page will be built properly by Sphinx